nym-wallet/src/context/bonding.tsx

```diff
@@ -78,6 +79,10 @@ export interface TBondedGateway {
   mixPort: number;
   verlocPort: number;
   version: string;
+  routingScore: {
+    current: number;
+    average: number;
+  };
```

Comment:
As the request to fetch routing data could potentially crash/or returns an error for some reasons, a better typing would be to make it nullable :  `routingScore?: ...`